components/infobox/commons/infobox_league.lua

```diff
@@ -240,15 +240,7 @@ function League:_setLpdbData(args)
 			sponsor5 = args.sponsor5,
 		}),
 		links = mw.ext.LiquipediaDB.lpdb_create_json({
-			discord = Links.makeFullLink('discord', args.discord),
-			facebook = Links.makeFullLink('facebook', args.facebook),
-			instagram = Links.makeFullLink('instagram', args.instagram),
-			twitch = Links.makeFullLink('twitch', args.twitch),
-			twitter = Links.makeFullLink('twitter', args.twitter),
-			website = Links.makeFullLink('website', args.website),
-			weibo = Links.makeFullLink('weibo', args.weibo),
-			vk = Links.makeFullLink('vk', args.vk),
-			youtube = Links.makeFullLink('youtube', args.youtube),
+			Links.makeFullLinksForTableItems(links)
```

Comment:
Doesn't this return a table? Should it be:

```lua
		links = mw.ext.LiquipediaDB.lpdb_create_json(Links.makeFullLinksForTableItems(links)),
```

Reply:
the change is inside the `mw.ext.LiquipediaDB.lpdb_create_json`
line 242: `links = mw.ext.LiquipediaDB.lpdb_create_json({`
now the changed stuff
line after that (243/252): `}),`

Reply:
Yes, but you're returning a table, inside a table

Reply:
right, my bad